x/did/types/auth_test.go

```diff
@@ -0,0 +1,68 @@
+package types
```

Comment:
This test can also be caught in a circular reference, so use the default 'suite'.

Reply:
Yeah. But, I think this test doesn't need to use the test suite actually, because it doesn't require any keepers.